src/modules/include-exclude/includes/Plugin_Enabled.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * This file contains the core plugin functions.
+ *
+ * @package Wordlift
+ */
+
+namespace WordLift\Modules\Include_Exclude;
+
+/**
+ * Define the JSON-LD Enabled class.
+ *
+ * The Core class instance will hook the `wl_jsonld_enabled` filter.
+ *
+ * @package Wordlift
+ */
+class Plugin_Enabled {
+
+	/**
+	 * Register hooks.
+	 */
+	public function register_hooks() {
+		add_filter( 'wl_is_enabled', array( $this, 'wl_is_enabled' ) );
+		add_filter( 'wl_jsonld_term_html_output', array( $this, 'may_be_exclude_nitropack' ) );
+		add_filter( 'wl_jsonld_post_html_output', array( $this, 'may_be_exclude_nitropack' ) );
```

Comment:
This doesn't look something that should be here, but more like a Nitropack adapter.